Include/internal/pycore_freelist.h

```diff
@@ -125,6 +125,16 @@ typedef struct _Py_freelist_state {
     struct _Py_object_stack_state object_stacks;
 } _PyFreeListState;
 
+extern void _PyObject_ClearFreeLists(_PyFreeListState *state, int is_finalization);
```

Comment:
Since this API is not a domain-specific API, I located all ClearFreeList API into here :)